app/src/main/res/layout/app_bar.xml

```diff
@@ -28,15 +28,15 @@
         <View
             android:layout_width="1dp"
             android:layout_height="match_parent"
-            android:background="?attr/colorPrimary" />
+            android:background="?attr/white" />
```

Comment:
This parameter value can not be found during the build and will fail the build. You can just leave the original value unchanged